node/hyperbahn/handler.js

```diff
@@ -178,9 +195,20 @@ function sendAdvertise(services, options, callback) {
 
     res: {}
 */
+
 HyperbahnHandler.prototype.handleRelayAdvertise =
 function handleRelayAdvertise(self, req, arg2, arg3, cb) {
-    /*eslint max-params: [2, 5]*/
+    self.handleRelay(req, arg2, arg3, cb, self.advertise);
```

Comment:
I see, what do you suggest doing here? I want to share most of the code for advertise and unadvertise.